playbooks/besman-sast-sonarqube-playbook.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+# Author: Samir Ranjan Parhi
+# License: Same as Repository Licence
+# Usage : Beta-Release
+# Date : 12/04/2024
+
+function __besman_init() {
+    __besman_echo_white "initialising"
+    export ASSESSMENT_TOOL_NAME="sonarqube"
+    export ASSESSMENT_TOOL_TYPE="SAST"
+    export ASSESSMENT_TOOL_VERSION="9.9.4"
+    export ASSESSMENT_TOOL_PLAYBOOK="besman-$ASSESSMENT_TOOL_TYPE-$ASSESSMENT_TOOL_NAME-playbook.sh"
```

Comment:
"besman-$ASSESSMENT_TOOL_TYPE-$ASSESSMENT_TOOL_NAME-playbook.sh" -> "besman-$ASSESSMENT_TOOL_TYPE-0.0.1-playbook.sh" (0.0.1- version of the playbook)

Reply:
@asa1997 : I am Sorry, But I will not do any changes unless the above query are addressed